source/neuropod/backends/torchscript/torch_backend.cc

```diff
@@ -433,6 +431,7 @@ std::unique_ptr<NeuropodValueMap> TorchNeuropodBackend::infer_internal(const Neu
         if (is_named_tuple)
         {
             // This is a named tuple
+            // NOLINTNEXTLINE(modernize-loop-convert): Can't always use a range based loop here
```

Comment:
As you mentioned, with setting 31 instead of 16bit, we should ask clang-tidy to catch the problem like the one below https://clang.llvm.org/extra/clang-tidy/checks/bugprone-too-small-loop-variable.html

Reply:
I may enable that in a follow up PR, but the issue will likely be taken care of by some additional compile time warnings I'm enabling.

(For anyone reading, see my comment here for more details on this `clang-tidy` check: https://github.com/uber/neuropod/pull/433#issuecomment-693768830)